client/src/components/questions/QuestionList.jsx

```diff
@@ -26,6 +26,8 @@ const QuestionList = ({ questions, productName, productID }) => {
   const [list, setList] = useState(questions);
   const [questionModal, setQuestionModal] = useState(false);
 
+  var count = 100;
```

Comment:
what is this variable for?

Reply:
it is for keeping track of test ids for each question . I can probably refactor this out at this point though.